src/website/app/Footer.js

```diff
@@ -26,10 +26,13 @@ const Root = createStyledComponent('div', ({ theme }) => ({
   color: theme.color_caption,
   fontSize: theme.fontSize_mouse,
   marginTop: getNormalizedValue(
-    `${parseFloat(theme.spacing_quad) * 2}em`,
+    `${parseFloat(theme.space_stack_sm) * 8}em`,
+    theme.fontSize_mouse
```

Comment:
I'm having a hard time reasoning about the changes to the inputs of getNormalizedValue. I get why we're using space_stack, but why are we using getNormalizedValue() instead of pxToEms()?

Also why are we multiplying by 8 instead of 2?

Finally, getNormalizedValue is going to parseFloat also, so we probably don't need to tack `em` on the end of line 29 or 33.

Reply:
> why are we using getNormalizedValue() instead of pxToEms()?

Because the former outputs an `em` value to be relative to the applied fontSize (`theme.fontSize_mouse`), while the latter does so relative to the base fontSize.

> Also why are we multiplying by 8 instead of 2?

Generally, if I needed an arbitrary multiplier like that, I refactored so that we're doing so on the "base unit" (8px), which is `theme.space_xx_sm`.

> Finally, getNormalizedValue is going to parseFloat also, so we probably don't need to tack em on the end of line 29 or 33.

You're right. It still expects a string, though, so I went ahead and added it for clarity.

Reply:
Agreed that `pxToEms` and `getNormalizedValue` are not very intuitive, nor is the multiplication by seemingly arbitrary values spread around everywhere. 

Perhaps the helper function names could be improved to make them more intuitive - maybe in another story?

Also, is the multiplication just to match the designs and because we don't have a spacing variable for that value?